api/emailRouter.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express");
+const validate = require("../validator/validator");
+const { verifyEmail } = require("../controller/email/verifyEmail");
+
+const { resendEmail } = require("../controller/email/resendEmail")
+const routerEmail = express.Router();
+
+
```

Comment:
```suggestion
```
Jedna pusta linia wystarczy do czytelności. A jeżeli dwie to dlaczego dwie? Potem trzeba pamietac w jakich wypadkach dajesz jedna a w jakich dwie.